modules/ROOT/pages/bat-write-tests-task.adoc

```diff
@@ -9,25 +9,37 @@ A test suite is composed of the following elements:
 
 Tests are written in the BDD Test-Writing syntax and use the `.dwl` file extension.
 
-The best way to learn how to write tests in this syntax is to use the BAT Playground, a web application that runs modifiable tests on the Deck of Cards API by default. There are nine different tests that are presented in ascending order of complexity. Each test includes comments to show you what the code does.
-
 [[before-you-begin]]
 == Before You Begin
 
-Before you can run the BAT playground, you must have access to an Anypoint Platform account. 
-
-* If you already have access to an Anypoint Platform account, proceed to <<start-bat-playground>>.
-
-* If you do not have access to an Anypoint Platform account, refer to xref::access-management::managing-your-account[Managing your Anypoint Platform Account] to find out how to sign up for a new account or get access to an organization in an existing account. After you have access to an Anypoint Platform account, proceed to <<start-bat-playground>>.
-
-[[start-bat-playground]]
-== Start the BAT Playground
-
-Start the BAT Playground by going to the following URL: https://anypoint.mulesoft.com/apitesting/playground/. You are prompted to log in to your Anypoint Platform account to access the BAT playground.
-
-After you learn the basics of writing tests from the examples in the BAT Playground, you can refer to the BDD Test-Writing Syntax Reference, which can help you go beyond the basics.
+Before you can write and run tests, you must have access to an Anypoint Platform account. 
+
+[[test-examples]]
+== Use Examples to Learn the Basics
+Learn the basics of writing tests using the provided examples. 
+
+To select examples:
+
+. In the *Create Monitor* or *Edit Monitor* page, click *Switch to code editor*. 
```

Comment:
Examples will be available only in the creation process, so we should remove the "or *Edit Monitor*" from this line

Reply:
Thanks for clarifying. So they can only see these for a new monitor, not an update?

Reply:
yes, just for new monitors